openstack/monasca/templates/etc/monasca-agent/conf.d/_prometheus.yaml.tpl

```diff
@@ -174,3 +174,5 @@ instances:
                dimensions:
                    cluster: cluster
                    service: service
+           swift:
+               gauges: [ 'swift_(async_pendings)_rate' ]
```

Comment:
You need to add the `kubernetes_namespace` to the `match_labels` field near line 126.